modules/default/weather/weather.js

```diff
@@ -124,6 +127,9 @@ Module.register("weather", {
 
 	// Select the template depending on the display type.
 	getTemplate: function () {
+		if (this.config.type === "daily") {
```

Comment:
maybe refactor it into a switch statement? and also add toLowerCase just in case.